### DISCUSSION TITLE
Welcome to yapCAD Discussions!

### DISCUSSION THREAD

**Discussion by rdevaul** [rdevaul/yapCAD#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GJoe2** [comment]
I had issues at building yapcad docs (Ubuntu):

1- After cloning the repo, I was about to compile the docs with  `make -C docs html` but I got:
`ModuleNotFoundError: No module named 'sphinx_rtd_theme'`
Which can be solved with:
 `pip3 install sphinx_rtd_theme`

2- After downloading the dependencies I managed to build the docs but I got various warnings like:

```
WARNING: autodoc: failed to import module 'combine' from module 'yapcad'; the following exception was raised:
No module named 'mpmath'
```

It managed to build the html file it was missing information in module section 
And it could be resolved installing the next packages:

`pip3 install mpmath ezdxf pyglet`


Probably the online documentation should be built considering the dependencies I addressed
